### PR TITLE
repeatedpassword

### PR DESCRIPTION
**Pull request by fabianmueller7**

### CODE REVIEW

File: src/app/register/register.component.ts [hunk 1]
```diff
@@ -17,14 +17,19 @@ export class RegisterComponent {
...
+<<<<<<< HEAD
     try {
       await this.authServie.createAccount(username, email, password);
       this.router.navigate(['']);
     } catch (error: any) {
       this.toasterService.showToast('error', error.message);
     }
+=======
+
+    await this.authServie.createAccount(username, email, password, repeatedpassword);
```
Comment (DeNic0la) [inline src/app/register/register.component.ts:31]: What the actual fuck

File: src/app/register/register.component.ts [hunk 2]
```diff
@@ -17,14 +17,19 @@ export class RegisterComponent {
...
+>>>>>>> dcde115 (repeatedpassword)
```
Comment (DeNic0la) [inline src/app/register/register.component.ts:32]: LE was ?

File: src/app/register/register.component.html [hunk 3]
```diff
@@ -30,10 +30,28 @@ <h2 class="title">Register</h2>
       #password
     ></app-textfield>
   </div>
+  <div class="label pt-3 flex flex-column align-items-start">
+    <label class="text" for="password">Repeat Password:</label>
+    <app-textfield
+      type="password"
+      id="password"
+      name="password"
```
Comment (DeNic0la) [inline src/app/register/register.component.html:38]: ehhhhmmmmmmmm

**Review by DeNic0la — CHANGES_REQUESTED** [review]
it doesnt even work

**Review by DeNic0la — CHANGES_REQUESTED** [review]
repeat password input field still has a fucking terrible id and name

**Review by DeNic0la — APPROVED** [review]
LGTM

**Review by DawidKapka — APPROVED** [review]

**Pull request merged**